CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
 # ember-cli Changelog
 
+### 1.13.0
+
+The following changes are required if you are upgrading from the previous
+version:
+
+- Users
+  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + `Brocfile.js` has been deprecated in favor of `ember-cli-build.js`. See [TRANSITION.md](https://github.com/ember-cli/ember-cli/blob/master/TRANSITION.md) for details on how to transition your `Brocfile.js` code to `ember-cli-build.js`.
+  + Components are now generated with integration tests by default instead of unit tests. Component unit tests can still be generated separately with: `ember g component-test foo-bar -unit`.
+  + Services can now be generated into pod structure.
+- Addon Developers
+  + `Brocfile.js` has been deprecated in favor of `ember-cli-build.js`. See [TRANSITION.md](https://github.com/ember-cli/ember-cli/blob/master/TRANSITION.md) for details on how to transition your `Brocfile.js` code to `ember-cli-build.js`.
+  + Blueprints can now be generated into the `tests/dummy/app` folder with the `--dummy` flag.
+  + Scoped npm dependencies are now supported.
+- Core Contributors
+  + fs.existsSync is deprecated, use exists-sync instead.
+
+#### Community Contributions
+- [#4391](https://github.com/ember-cli/ember-cli/pull/4391) Update ember-data to 1.13.4
+- [#4300](https://github.com/ember-cli/ember-cli/pull/4300) Update Ember to v1.13.1. [@rwjblue](https://github.com/rwjblue)
```

Comment:
Latest ember version? We're on [1.13.3](https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/bower.json#L4)